api-module-library/slack/api.js

```diff
@@ -101,7 +101,8 @@ class Api extends OAuth2Requester {
         } else if (
             parsedResponse.error === 'invalid_auth' ||
             parsedResponse.error === 'auth_expired' ||
-            parsedResponse.error === 'token_expired'
+            parsedResponse.error === 'token_expired' ||
+            parsedResponse.error === 'token_revoked'
```

Comment:
Only thought... `token_revoked`, would definitely not be refreshable. But, doesn't hurt to retry and fail